content/channels/index.textile

```diff
@@ -447,6 +447,72 @@ To publish a single message to multiple channels, make multiple "@publish()@":/a
+_*Solution:*_ Server-side batching can combine multiple reactions into a batch before sending them to subscribers. This helps manage the high volume of interactions efficiently, preventing rate limits from being exceeded and ensuring all reactions are delivered promptly.
+
+* *Cursor tracking:*
+A collaborative workspace application tracks user presence and cursor movements in realtime, showing the live cursor positions of multiple users. Continuous updates on cursor positions from numerous users can generate a high volume of messages, increasing costs and potentially hitting rate limits.
+_*Solution:*_ The application can batch these frequent updates into single messages by applying server-side batching to the cursor and presence messages. This reduces the number of messages processed and sent, lowering costs and maintaining realtime tracking accuracy.
+
+h4. How server-side batching works
+
+The server-side batching process works through a *batching policy* and a *batching interval:*
+
+h4. Batching policy
+
+The batching policy defines the rules and behaviors for aggregating messages during the batching interval. The initial release supports the *simple* policy, which groups multiple incoming messages into a single batch without altering their content or metadata.
+
+A batch can contain up to 50 messages by count or total data size. For example, if you have 60 messages, they will be split into two batches: one with 50 messages and another with 10. If the combined data size of 50 messages exceeds a specific bandwidth threshold _e.g. in kilobytes_ the batch size will be limited to that threshold.
+
+h4. Batching interval
+
+The batching interval is a configurable period specified in milliseconds _ranging from 20ms to 1s_, during which incoming messages are collected and grouped into a batch. Messages sent to the server during this interval are temporarily held and aggregated. Once the interval elapses, the collected messages are combined into a single batch and delivered to subscribers as one message.
+
+h4. Choose the right interval for your app
+
+* Shorter intervals benefit real-time applications.
+* Longer intervals are preferred for cost efficiency.
+* Align the interval with the expected user experience.
+
+h4. Configure server-side batching
+
+The following steps take you through the server-side configuration process:
+
+# On your "dashboard":ably.com/accounts/40946, select one of your apps.
+# Go to *settings*.
+# Under channel rules, add a *new rule*.
+# Enter the *namespace* or *channel ID* to which you want to add a new rule.
+# Select *server-side batching enabled*.
+# Choose a batching policy (only *simple* is available in beta).
+# Choose an interval range value from 20ms to 1s.
+# Click *create* channel rule to save.
```

Comment:
The rule enablement side should probably link to the channel rules section on this page that covers it. We should also add batching to the table in that section.